site3/website/src/pages/release-notes.md

```diff
@@ -1,6 +1,99 @@
+* Fix garbage collection blocked by runtime exception [PR #3901](https://github.com/apache/bookkeeper/pull/3901) 
+* Skip sync the RocksDB when no changes [PR #3904](https://github.com/apache/bookkeeper/pull/3904) 
+* Enable PCBC completionObjects autoShrink to reduce memory usage and gc [PR #3913](https://github.com/apache/bookkeeper/pull/3913) 
+* [Fix] Recycle dropping read-write requests when various exceptions happened [PR #3912](https://github.com/apache/bookkeeper/pull/3912) 
+* Fix isWritable incorrect problem. [PR #3918](https://github.com/apache/bookkeeper/pull/3918) 
+* Fix ledger replicated failed blocks bookie decommission process [PR #3917](https://github.com/apache/bookkeeper/pull/3917) 
+* Support skip invalid journal record in replying journal stage [PR #3956](https://github.com/apache/bookkeeper/pull/3956) 
+* Avoid compaction to trigger extra flushes DbLedgerStorage [PR #3959](https://github.com/apache/bookkeeper/pull/3959) 
+* Fix memory leak of direct memory in direct memory entry logger. [PR #3983](https://github.com/apache/bookkeeper/pull/3983) 
+* Unify ByteBufAllocator for the DirectIO component [PR #3985](https://github.com/apache/bookkeeper/pull/3985) 
+* Print compaction progress [PR #4071](https://github.com/apache/bookkeeper/pull/4071) 
+* Optimize bookie decommission check wait interval [PR #4070](https://github.com/apache/bookkeeper/pull/4070) 
+* Fix trigger GC not work [PR #3998](https://github.com/apache/bookkeeper/pull/3998) 
+* Allow to set max operation numbers in a single rocksdb batch [PR #4044](https://github.com/apache/bookkeeper/pull/4044) 
+* Add read failed log for ledger checker. [PR #4010](https://github.com/apache/bookkeeper/pull/4010) 
+* Fix read write request leak when executor throw `RejectedExecutionException` [PR #4024](https://github.com/apache/bookkeeper/pull/4024) 
+* Improve DefaultEntryLogger read performance. [PR #4038](https://github.com/apache/bookkeeper/pull/4038) 
+
+#### Client
+
+* [Bug] Always one orphan ledger is created [PR #3813](https://github.com/apache/bookkeeper/pull/3813) 
+* Fix checksum calculation bug when the payload is a CompositeByteBuf with readerIndex > 0 [PR #4196](https://github.com/apache/bookkeeper/pull/4196) 
+* Fix no known bookies after reset racks for all BKs [PR #4128](https://github.com/apache/bookkeeper/pull/4128) 
+* Fix issue with binary compatibility with older grpc versions at runtime in the client [PR #3997](https://github.com/apache/bookkeeper/pull/3997) 
+* Entry write support local node region aware placement policy [PR #4063](https://github.com/apache/bookkeeper/pull/4063) 
+* Rackaware placement policy support local node awareness by hostname [PR #4057](https://github.com/apache/bookkeeper/pull/4057) 
+* Use netty-bom for aligning netty library versions, add epoll for linux-aarch_64 [PR #4204](https://github.com/apache/bookkeeper/pull/4204) 
+
+#### AutoRecovery
```

Comment:
Please add this PR into the auto recovery section https://github.com/apache/bookkeeper/pull/4211